### ISSUE TITLE
Release started failing with v0.13.0

### ISSUE THREAD

**Issue by cjellick** [goreleaser/goreleaser#3599]
### What happened?

We cut our latest release of https://github.com/acorn-io/acorn and it failed with the following error:
```
  ⨯ release failed after 3m15s               error=signing docker images: failed to publish artifacts: sign: cosign failed: exit status 1: Error: signing [ghcr.io/acorn-io/acorn:v0.4.1-rc1@]: parsing reference: could not parse reference: ghcr.io/acorn-io/acorn:v0.4.1-rc1@
main.go:46: error during command execution: signing [ghcr.io/acorn-io/acorn:v0.4.1-rc1@]: parsing reference: could not parse reference: ghcr.io/acorn-io/acorn:v0.4.1-rc1@
````

Our releases were setup to use the "latest" version of goreleaser and this was v1.13.0. When I pinned goreleaser to v1.12.3, the release worked again.

### How can we reproduce this?

I don't have a step-by-step way to repro, but this is our .goreleaser file:
https://github.com/acorn-io/acorn/blob/main/.goreleaser.yml

This is how we kick it off from github workflow:
https://github.com/acorn-io/acorn/blob/main/.github/workflows/release.yaml#L46-L63

### goreleaser version

```bash
Since this happened from a github workflow, I cannot run this command, but the build logs clearly indicate v1.13.0 was being used:

Downloading https://github.com/goreleaser/goreleaser/releases/download/v1.13.0/goreleaser_Darwin_all.tar.gz
Extracting GoReleaser
/usr/bin/tar xz -C /Users/darren/actions-runner/_work/_temp/2e4968d0-e5f3-4d66-89f2-4c47a0b6a72c -f /Users/darren/actions-runner/_work/_temp/32225a7f-58cc-4f70-b9ec-2559a3cca85b
GoReleaser latest installed successfully
```

### GoReleaser Check

- [X] goreleaser check shows no errors

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

Here's a slightly bigger chunk of the build logs, in case it helps:
```
generating changelog
    • writing                                        changelog=releases/CHANGELOG.md
    • took: 1s
  • archives
    • creating                                       archive=releases/acorn-v0.4.1-rc1-linux-arm64.tar.gz
    • creating                                       archive=releases/acorn-v0.4.1-rc1-windows-amd64.zip
    • creating                                       archive=releases/acorn-v0.4.1-rc1-linux-amd64.tar.gz
    • creating                                       archive=releases/acorn-v0.4.1-rc1-macOS-universal.tar.gz
    • creating                                       archive=releases/acorn-v0.4.1-rc1-linux-arm7.tar.gz
    • took: 13s
  • calculating checksums
  • signing artifacts
    • refreshing checksums                           file=checksums.txt
    • signing                                        artifact=checksums.txt cmd=cosign signature=releases/checksums.txt.sig
    • refreshing checksums                           file=checksums.txt
    • took: 3s
  • homebrew tap formula
    • guessing install to be "bin.install \"acorn\""
    • guessing install to be "bin.install \"acorn\""
    • guessing install to be "bin.install \"acorn\""
    • writing                                        formula=releases/acorn.rb
  • docker images
    • building docker image                          image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm32v7
    • building docker image                          image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-amd64
    • building docker image                          image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm64
    • took: 17s
  • publishing
    • docker images
      • pushing                                      image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm64
      • pushing                                      image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-amd64
      • pushing                                      image=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm32v7
      • took: 19s
    • docker manifests
      • creating                                     images=[ghcr.io/acorn-io/acorn:v0.4.1-rc1-amd64@sha256:3361513e269494a1b30a94e0c69ffc838604dc4e2d3fac63ff34593b63f59c19 ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm64@sha256:52cb57d851602c8680a32d46fcd705a10d7ac155c13101a24a3[74](https://github.com/acorn-io/acorn/actions/runs/3567762373/jobs/5996478509#step:10:75)3c533dec999 ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm32v7@sha256:cac2d3cf85a8e5bcb0ddabdc2c331f509ec78b10072509e830a0862e58d6b040] manifest=ghcr.io/acorn-io/acorn:v0.4.1-rc1
      • pushing                                      manifest=ghcr.io/acorn-io/acorn:v0.4.1-rc1
      • creating                                     images=[ghcr.io/acorn-io/acorn:v0.4.1-rc1-amd64@sha256:3361513e269494a1b30a94e0c69ffc838604dc4e2d3fac63ff34593b63f59c19 ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm64@sha256:52cb57d851602c8680a32d46fcd705a10d7ac155c13101a24a3743c533dec999 ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm32v7@sha256:cac2d3cf85a8e5bcb0ddabdc2c331f509ec78b10072509e830a0862e58d6b040] manifest=ghcr.io/acorn-io/acorn:latest
      • pushing                                      manifest=ghcr.io/acorn-io/acorn:latest
      • took: 13s
    • signing docker images
      • signing                                      artifact=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm64 cmd=cosign
      • signing                                      artifact=ghcr.io/acorn-io/acorn:v0.4.1-rc1-amd64 cmd=cosign
      • signing                                      artifact=ghcr.io/acorn-io/acorn:v0.4.1-rc1-arm32v7 cmd=cosign
      • signing                                      artifact=ghcr.io/acorn-io/acorn:v0.4.1-rc1 cmd=cosign
      • took: 11s
  • took: 43s
  ⨯ release failed after 3m15s               error=signing docker images: failed to publish artifacts: sign: cosign failed: exit status 1: Error: signing [ghcr.io/acorn-io/acorn:v0.4.1-rc1@]: parsing reference: could not parse reference: ghcr.io/acorn-io/acorn:v0.4.1-rc1@
main.go:46: error during command execution: signing [ghcr.io/acorn-io/acorn:v0.4.1-rc1@]: parsing reference: could not parse reference: ghcr.io/acorn-io/acorn:v0.4.1-rc1@
```

**Comment by blacktop** [comment]
I'm also seeing weird failures:

<details>
<summary>error output</summary>
<br>

```bash
      • uploading to release                         file=dist/ipsw_3.1.200_x86_64.apk name=ipsw_3.1.200_x86_64.apk
      • uploading to release                         file=dist/ipsw-3.1.200.x86_64.rpm name=ipsw-3.1.200.x86_64.rpm
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • uploading to release                         file=dist/ipsw_3.1.200_amd64.deb name=ipsw_3.1.200_amd64.deb
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • uploading to release                         file=dist/ipsw-3.1.200-1-aarch64.pkg.tar.zst name=ipsw-3.1.200-1-aarch64.pkg.tar.zst
      • uploading to release                         file=dist/ipsw-3.1.200-1-x86_64.pkg.tar.zst name=ipsw-3.1.200-1-x86_64.pkg.tar.zst
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • uploading to release                         file=dist/ipsw-3.1.200-1-aarch64.pkg.tar.zst name=ipsw-3.1.200-1-aarch64.pkg.tar.zst
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_windows_arm64.tar.gz.sbom name=ipsw_3.1.200_windows_arm64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_windows_x86_64.tar.gz.sbom name=ipsw_3.1.200_windows_x86_64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_macOS_arm64.tar.gz.sbom name=ipsw_3.1.200_macOS_arm64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_ios_arm64.tar.gz.sbom name=ipsw_3.1.200_ios_arm64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_macOS_x86_64.tar.gz.sbom name=ipsw_3.1.200_macOS_x86_64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_linux_arm64.tar.gz.sbom name=ipsw_3.1.200_linux_arm64.tar.gz.sbom
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_linux_x86_64.tar.gz.sbom name=ipsw_3.1.200_linux_x86_64.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_macOS_arm64_extras.tar.gz.sbom name=ipsw_3.1.200_macOS_arm64_extras.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_macOS_x86_64_extras.tar.gz.sbom name=ipsw_3.1.200_macOS_x86_64_extras.tar.gz.sbom
      • uploading to release                         file=/Users/blacktop/Developer/Github/blacktop/ipsw/dist/ipsw_3.1.200_macOS_universal.tar.gz.sbom name=ipsw_3.1.200_macOS_universal.tar.gz.sbom
      • uploading to release                         file=dist/checksums.txt name=checksums.txt
      • uploading to release                         file=dist/checksums.txt.sig name=checksums.txt.sig
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64280->140.82.112.13:443: use of closed network connection try=2
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64281->140.82.112.13:443: use of closed network connection try=3
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64282->140.82.112.13:443: write: broken pipe try=4
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 404 Not Found [] try=2
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64291->140.82.113.14:443: use of closed network connection try=5
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64292->140.82.114.13:443: use of closed network connection try=6
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 404 Not Found [] try=2
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=2
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64278->140.82.112.13:443: use of closed network connection try=7
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200-1-aarch64.pkg.tar.zst error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200-1-aarch64.pkg.tar.zst: 404 Not Found [] try=1
      • uploading to release                         file=dist/ipsw-3.1.200-1-aarch64.pkg.tar.zst name=ipsw-3.1.200-1-aarch64.pkg.tar.zst
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=Post "https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk": write tcp 192.168.4.191:64294->140.82.114.14:443: use of closed network connection try=8
      • uploading to release                         file=dist/ipsw_3.1.200_aarch64.apk name=ipsw_3.1.200_aarch64.apk
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200-1-aarch64.pkg.tar.zst error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200-1-aarch64.pkg.tar.zst: 422 Validation Failed [{Resource:ReleaseAsset Field:name Code:already_exists Message:}] try=2
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_aarch64.apk error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_aarch64.apk: 422 Validation Failed [{Resource:ReleaseAsset Field:name Code:already_exists Message:}] try=9
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=2
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 404 Not Found [] try=3
      • uploading to release                         file=dist/ipsw-3.1.200.aarch64.rpm name=ipsw-3.1.200.aarch64.rpm
      • failed to upload artifact, will retry        artifact=ipsw-3.1.200.aarch64.rpm error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200.aarch64.rpm: 422 Validation Failed [{Resource:ReleaseAsset Field:name Code:already_exists Message:}] try=4
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=3
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=3
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=4
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=4
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=5
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=5
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=6
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=6
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=7
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 404 Not Found [] try=7
      • uploading to release                         file=dist/ipsw_3.1.200_arm64.deb name=ipsw_3.1.200_arm64.deb
      • failed to upload artifact, will retry        artifact=ipsw_3.1.200_arm64.deb error=POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw_3.1.200_arm64.deb: 422 Validation Failed [{Resource:ReleaseAsset Field:name Code:already_exists Message:}] try=8
      • took: 2m41s
  • took: 3m39s
  ⨯ release failed after 4m17s               error=scm releases: failed to publish artifacts: failed to upload ipsw-3.1.200-1-aarch64.pkg.tar.zst after 2 tries: POST https://uploads.github.com/repos/blacktop/ipsw/releases/84378518/assets?name=ipsw-3.1.200-1-aarch64.pkg.tar.zst: 422 Validation Failed [{Resource:ReleaseAsset Field:name Code:already_exists Message:}]
make: *** [release] Error 1
```

</details>

**Comment by caarlos0** [comment]
@blacktop that's unrelated to this issue, there are a few other reports of this ranging from a couple of years ago to now... its always either user error or a github bug that's impossible to reliably reproduce. Please search the issues for more info.

**Comment by caarlos0** [comment]
@cjellick found the bug, you can avoid if for now by changing your signing config:


```yaml
docker_signs:
  - artifacts: all
    stdin: "{{ .Env.COSIGN_PASSWORD }}"
    args: ["sign", "--key=cosign.key", "${artifact}"]
```

(add the _args_ line)

meanwhile, I'm working on a fix.

**Comment by caarlos0** [comment]
okay, PR fixing it is opened, thanks for bringing this up @cjellick ! I'll release a patch after dinner :)

**Comment by cjellick** [comment]
awesome. Thanks so much!

**Comment by caarlos0** [comment]
Should be fixed in v1.13.1.

thanks again for reporting 🙏

**Comment by cjellick** [comment]
Thank you! Will upgrade today.
